crates/wasmtime/src/runtime/types.rs

```diff
@@ -1617,6 +1618,7 @@ impl StructType {
         Self::from_wasm_struct_type(
             engine,
             finality.is_final(),
+            false, // TODO: handle shared
```

Comment:
In this file I think it's fine to just set `shared: false` at the end. There'll be more plumbing eventually to handle this if needed but for now it's an accurate reflection that only unshared types are created here.